osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -121,21 +119,33 @@ private void load()
 
         protected override void LoadComplete()
         {
-            Sort.BindValueChanged(onSortChanged, true);
+            Sort.BindValueChanged(_ => refetchComments());
```

Comment:
why is the `true` gone?

Reply:
No reason to call it right away because we have no parameters anyway, I guess?

Reply:
what?

Reply:
I mean, we have no reason to trigger `refetchComments() ` on load completed if we have no type nor id to use (I hope I explained it right now)

Reply:
aren't you making this to be a usable component?

Reply:
true was there because we already have all the parameters in place (passed in the ctor) and we wanted to fetch comments right away when the container has been loaded. Now we don't have any of these so I removed it.

Reply:
So what happens if the `Show()` method is called before `LoadComplete()`?

Reply:
Crash due to not loaded api

Reply:
I see: we can add `return` if not loaded and add true here, thanks for good example